.github/workflows/lint-code.yml

```diff
@@ -85,3 +85,19 @@ jobs:
         with:
           isortVersion: "latest"
           requirementsFiles: "requirements.txt requirements-dev.txt"
+
+  mypy:
```

Comment:
We could also add it to pre-commit :) 
Basically adding:
```
  - repo: https://github.com/pre-commit/mirrors-mypy
    rev: '105'
    hooks:
    -   id: mypy
```
to `.pre-commit-config.yaml`

Reply:
All in favor of it, but this would't make the action obsolete. Or would it?

Reply:
no, they can coexist 🙂